Sprint 19 (home work) Gr26

amanzhola opened the pull request.

amanzhola committed:
Sprint 19 (home work) Gr26
1 -> MainActivity(partial transfer to Jetpack Navigation Component)
➕
MediaLibraryFragment
SettingsFragment
SearchFragment

Done temp compromise between Fragment and Activity for BottomNavigationHelper to have final transfer to Jetpack Navigation Component
Toolbar survived to have functions for fragments also as for activities

additional updates:
2 -> 💿📚📀MediaLibrary ➕ FragmentFavouriteTracks(🅰️❤️🎶) and FragmentPlaylist(🅱️🧾🎶) get back lost on transfer to fragments bottom navigation

3 -> MovieRootActivity (full transfer to Jetpack Navigation Component) 👇

1 -> ✨📽️💃Movie 👉 MovieRootActivity
➕
 AboutFragment(📝🖼️) and PosterFragment(🧑‍🤝‍🧑‍🖼️)
➕
MoviesCastFragment
DetailsFragment
NamesFragment

1.1 -> ⭐ свайп с левого края вправо выход ➡️ в список SearchMovie, остальные свайпы стандарнт

amanzhola committed:
Sprint 19 (home work) Gr26
1 -> MainActivity(partial transfer to Jetpack Navigation Component)
➕
MediaLibraryFragment
SettingsFragment
SearchFragment

Done temp compromise between Fragment and Activity for BottomNavigationHelper to have final transfer to Jetpack Navigation Component
Toolbar survived to have functions for fragments also as for activities

additional updates:
2 -> 💿📚📀MediaLibrary ➕ FragmentFavouriteTracks(🅰️❤️🎶) and FragmentPlaylist(🅱️🧾🎶) get back lost on transfer to fragments bottom navigation

3 -> MovieRootActivity (full transfer to Jetpack Navigation Component) 👇

1 -> ✨📽️💃Movie 👉 MovieRootActivity
➕
 AboutFragment(📝🖼️) and PosterFragment(🧑‍🤝‍🧑‍🖼️)
➕
MoviesCastFragment
DetailsFragment
NamesFragment

1.1 -> ⭐ свайп с левого края вправо выход ➡️ в список SearchMovie, остальные свайпы стандарнт

anastasianadelshtehel commented on app/src/main/java/com/example/playlistmaker/ui/moviePosters/DetailsActivity.kt:
```diff
@@ -1,117 +1,117 @@
+//import com.google.android.material.tabs.TabLayoutMediator
+//import org.koin.androidx.viewmodel.ext.android.viewModel
+//
+//class DetailsActivity : AppCompatActivity() {
```
Закомментированный код нужно удалять

anastasianadelshtehel reviewed (commented):
Нужно удалить экраны не относящиеся к проекту, также настроить навигацию, если мы переключаемся по разным экранам и потом нажимаем кнопку назад, нужно реализовать просто выход из приложения, а не показываться весь стек экранов, также нужно удалить главный экран с 3 кнопками
https://github.com/user-attachments/assets/5a3d6e0d-c432-49ee-9b74-e2156c76a7d9

anastasianadelshtehel commented:
свич переключения темы также не соответствует макету
<img width="1266" height="870" alt="image" src="https://github.com/user-attachments/assets/6e32f9fa-0092-42f7-8258-5f4df4502dea" />

amanzhola committed:
Sprint 19 (home work) Gr26
1 -> При нажатии на Settings убраны все переходы кроме 3х экранов
2 -> очищен код от комментов насколько можно
3 -> Цвет на кнопку свича в Settings в day mode выделил явно для соответствия макету Фигма

Доп инфо:
В проекте ничего нет имеющего какое-либо отношение к пэт проекту, все сделанное соотвествует 100% (домашним) заданиям Практикума

amanzhola committed:
Sprint 19 (home work) Gr26

Сделано ->
✅ "Кнопка назад" = аппаратная (системная) кнопка "Назад"
Или жест «назад», если на устройстве нет кнопки.

📌 Закрывать приложение, а не возвращать на предыдущую вкладку.
